packages/contracts-periphery/package.json

```diff
@@ -80,6 +80,7 @@
     "node-fetch": "^2.6.7",
     "prettier": "^2.8.0",
     "prettier-plugin-solidity": "^1.0.0-beta.18",
+    "rimraf": "^5.0.1",
```

Comment:
I think we might need this in the root package.json: 

<img width="428" alt="image" src="https://github.com/ethereum-optimism/optimism/assets/23033765/e7b872a2-f9b3-4dd0-b427-4f79b2c97832">

Reply:
Oh good call if we put it there it becomes global but in this case it's kinda useful if it's this commonly used

Reply:
Wait how tf did this ever work before???